splinter_vendor/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(splinter_vendor)
+
+find_package(ament_cmake REQUIRED)
+find_package(Eigen3 REQUIRED)
+
+ament_add_default_options()
```

Comment:
failing on https://github.com/osrf/buoy_msgs/runs/8256207995?check_suite_focus=true#step:4:3705
`Unknown CMake command "ament_add_default_options".`

Reply:
looks like it currently only sets:
```
option(
      BUILD_SHARED_LIBS
      "Global flag to cause add_library() to create shared libraries if on. \
       If set to true, this will cause all libraries to be built shared \
       unless the library was explicitly added as a static library."
      ON)
```